src/data_model.py

```diff
@@ -42,4 +42,11 @@ class Waypoint:
     Waypoints are positions where the drone should fly to and capture a photo.
     """
 
-    pass
+    def __init__(self, x, y, z, yaw=0):
+        self.x = x  # X coordinate of the waypoint (in meters)
+        self.y = y  # Y coordinate of the waypoint (in meters)
+        self.z = z  # Z coordinate of the waypoint (in meters)
+        self.yaw = yaw  # Yaw angle at the waypoint (in radians)
```

Comment:
Nice that you modeled yaw here! This is technically more accurate since the command for the drone has to be position as well as orientation.